docs/topics/settings.rst

```diff
@@ -742,6 +742,17 @@ If :setting:`DOWNLOAD_DELAY` is zero (default) this option has no effect.
 
 .. _wget: http://www.gnu.org/software/wget/manual/wget.html
 
+.. setting:: REACTOR_THREADPOOL_MAXSIZE
+
+REACTOR_THREADPOOL_MAXSIZE
+------------------
```

Comment:
The underline is too short - it'll break the markup. ReST requires underline length to be >= text length.